kubectl-rook-ceph.sh

```diff
@@ -477,7 +477,21 @@ function run_stop_debug() {
   KUBECTL_NS_CLUSTER scale deployments "$deployment_name" --replicas=1
 }
 
-function run_debug() {
+function parse_image_flag() {
+  local flag="$1"
+  local val="$2"
+  case "$flag" in
+  "--alternate-image")
+    val_exists "$val" || return 1 # val should exist
+    ALTERNATE_IMAGE="${val}"
```

Comment:
Actually `ALTERNATE_IMAGE` var needs to pass with this function `parse_image_flag` and this function is already a sub-function call from `  parse_flags parse_image_flag "$@"`  so I was not sure how we can also pass the alternate_image var, So that's why kept it Global